examples/fields.go

```diff
@@ -3,16 +3,17 @@ package main
 
 import (
 	"fmt"
+	"os"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
 )
 
 func main() {
 	groupService, err := service.NewNimbleGroupService(
-		service.WithHost("1.1.1.1"),
-		service.WithUser("xxx"),
-		service.WithPassword("xxx"))
+		service.WithHost(os.Getenv("GO_SDK_TEST_HOST")),
```

Comment:
For these examples with a main() function, is there a convenient way to invoke them with options?  Perhaps it would be appropriate to define flags to allow the values to be specified on invocation, and fall back on the environment variables if no options are given.